tests/functional/binary-cache.sh

```diff
@@ -14,9 +14,9 @@ clearStore
 clearCache
 outPath=$(nix-build dependencies.nix --no-out-link)
 
-nix copy --to file://$cacheDir $outPath
+nix copy --to file://"$cacheDir" "$outPath"
```

Comment:
I would include ``file://`` in the quote, too but I think that makes not difference.